MiniBank/settings.py

```diff
@@ -75,8 +76,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'MiniBank',
+        'USER': 'postgres',
```

Comment:
For user, host and port we need to get it from environment variables or you can use `os.getenv` and pass default values.